doc/bot.md

```diff
@@ -109,7 +109,7 @@ We'll be using tags, primarily, to help us understand what needs attention, what
   - Will use Squash merge strategy
   - Will attempt to delete branch after merge, if possible
   - Will automatically remove the `AutoMerge` label if changes are pushed by someone *without* Write Access.
-  - More information on bot-logic that can be controlled with comments is [here](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Advanced-auto-merge)
+  - See more [information on bot-logic that can be controlled with comments](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Advanced-auto-merge)
```

Comment:
This is one of the cases where the accessibility pattern complained (it's important for people using screen readers)

Reply:
I am *so excited* for having accessibility patterns. Seriously, this Action is a tour-de-force.